apps/twap/README.md

```diff
@@ -0,0 +1,7 @@
+# Awesome Project Build with TypeORM
+
+Steps to run this project:
+
+1. Run `npm i` command
+2. Setup database settings inside `data-source.ts` file
+3. Run `npm start` command
```

Comment:
`yarn`?

Reply:
Autogenerated filed. Replaced npm with yarn in general.